pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -0,0 +1,67 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/insomniacslk/dhcp/dhcpv6"
+	"github.com/insomniacslk/dhcp/dhcpv6/server6"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/log"
+)
+
+const (
+	infiniteLease = 999 * 24 * time.Hour
+)
+
+type DHCPv6Handler struct {
+	clientIP      net.IP
+	leaseDuration time.Duration
+}
+
+func SingleClientDHCPv6Server(
+	clientIP net.IP,
+	serverIface string,
+	dnsIPs [][]byte,
+	routes *[]netlink.Route,
+	searchDomains []string,
+	mtu uint16,
+	customDHCPOptions *v1.DHCPOptions) error {
+
+	log.Log.Info("Starting SingleClientDHCPv6Server")
+
+	handler := &DHCPv6Handler{
+		clientIP:      clientIP,
+		leaseDuration: infiniteLease,
+	}
+
+	s, err := server6.NewServer("", nil, handler.ServeDHCPv6)
+	if err != nil {
+		return fmt.Errorf("couldn't create dhcpv6 server: %v", err)
+	}
+	go func() {
```

Comment:
`SingleClientDHCPv6Server` is already called as a goroutine, why spawn another one here?

Reply:
good point. Wil remove the goroutine from here.